### PR TITLE
TilesList: Add role "row" around rows

### PR DESCRIPTION
**Pull request by erichdev**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Add a wrapper around TilesList rows to give them role="row". This is a portion of an accessibility fix to support aria-selected in TilesList.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12053)

### CODE REVIEW

**Comment by erichdev** [comment]
@ThomasMichon will do some additional work to complete accessibility fixes for TilesList.

File: packages/experiments/src/components/TilesList/TilesList.tsx [hunk 1]
```diff
@@ -327,6 +333,11 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
 
       const margin = grid.spacing / 2;
 
+      if (currentRowCells.length > 0) {
```
Comment (aneeshack4) [inline packages/experiments/src/components/TilesList/TilesList.tsx]: What is the purpose of this? Thought this PR was just to add the role

**Comment by aneeshack4** [comment]
Pinging @ThomasMichon as a reminder to review, it's been sitting here for a while

**Comment by ThomasMichon** [comment]
Closed this as it has now been replaced by #12521.

**Pull request closed without merging**